data.lua

```diff
@@ -179,3 +179,21 @@ directory{
 	source = "Google and Wikipedia",
 }
 
+data{
+	file = "caragua_classes2010_regioes.tif",
+	summary = "Social Classe 2010 Real.",
+	source = "Feitosa et. al (2014) URBIS-Caraguá: Um Modelo de Simulação Computacional para a Investigação de Dinâmicas de Ocupação Urbana em Caraguatatuba, SP.",
+	image = "urbis_2010_real.PNG",
+	attributes = {
+		["0"] = "Categorizes the social conditions of households in Caraguatatuba using classes A (3), B (2) or C (1)."
+	}
+}
+
+data{
+	file = "vegtype_2000_5880.tif",
```

Comment:
Add the following code as an example for the packag (name it `vegetation.lua`) in order to reproduce the problem in the comment in the last lines of the code. 

```lua
-- @example Implementation of a simple Application from INLAND model for Brazil.

import("publish")
import("gis")

proj = Project{
    file = "vegetation.tview",
    clean = true,
    vegtype = filePath("vegtype_2000_5880.tif", "publish")
}

app = Application{
    project = proj,
    description = "The data of this application were extracted from INLAND project (http://www.ccst.inpe.br/projetos/inland/).",
    output = "VegetationWebMap",
    clean = true,
    simplify = false,
    title = "Vegetation scenario",
    vegtype = View {
        title = "Vegetation Type 2000",
        description = "Vegetation type Inland.",
        select = "value", -- This argument should not be necessary for raster data, as it already knows the attribute to be created.
        -- In fact, when one set this argument when using a raster data, it should stop with an error.
        value = {-127, 1, 2, 3, 9, 10, 12},
        color = {"red", "blue", "green", "yellow", "brown", "cyan", "orange"}
    }
}
```